### ISSUE TITLE
Game not responding...?

### ISSUE THREAD

**Issue by TomaszBarowiec** [CreativeMD/ItemPhysic#145]
That`s an odd one... 
So first things first- When opening the word on minecraft 1.16.37 forge, when breaking blocks, items (wood, dirt etc.) does not pup up. Trying to hit animals, causes nothing to be happen- like playing on server and than somebody take out the computer LAN and you can see that every animal in the game is repeating the same animation in one place. When resetting the minecraft couple of times (exit and enter the word and exit from the game and open the word) makes no difference- everything is the same. Maybe it`s the cause of other 90 mods I have, but after uninstalling and load once again the two mods- item phisics mod and Phisics mod, game started to respond. 
So in summary if you take these two mods, or one of these mods, or just load different configuration of mods with these mods included, it can fix game...

**Comment by CreativeMD** [comment]
Can you send me the log file?

**Comment by CreativeMD** [comment]
I will close this issue for now. If it still doesn't work let me know here.
